File: Linguagem C Exercicios/C-Condicional/AUMENTO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double salario,novoSalario,aumento = 0;
    int porcentagem =0;

    printf("Digite o salario da pessoa: R$ ");
    scanf("%lf",&salario);

    if (salario <= 1000.00){
        porcentagem = 20;
        aumento = (salario * porcentagem) / 100;
        novoSalario = aumento + salario;
    }else if(salario <= 3000.00){
        porcentagem = 15;
        aumento = (salario*porcentagem) / 100;
        novoSalario = aumento + salario;
    }else if(salario <= 8000.00) {
        porcentagem = 10;
        aumento = (salario*porcentagem) / 100;
        novoSalario = aumento + salario;
    }else {
        porcentagem = 5;
        aumento = (salario*porcentagem) / 100;
        novoSalario = aumento + salario;
    };

    printf("Novo salario: R$ %.2lf\n",novoSalario);
    printf("Aumento: R$ %.2lf\n",aumento);
    printf("Porcentagem: %d%\n",porcentagem);


    return 0;
}

File: Linguagem Java/Exercicios/Aumento.java
import java.util.Locale;
import java.util.Scanner;

public class Aumento {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double salario,novoSalario,aumento;
		int porcentagem;
		
		System.out.print("Digite o salario da pessoa: ");
        salario = sc.nextDouble();
        
        if (salario <= 1000.00) {
        	porcentagem = 20;        	
        }else if (salario <= 3000.00) {
        	porcentagem = 15;        	
        }else if (salario <= 8000.00) {
        	porcentagem = 10;        	
        }else {
        	porcentagem = 5;        	
        };
        
        aumento = (salario / 100) * porcentagem;
        novoSalario = salario + aumento;
        
        System.out.println("Novo salario = "+String.format("%.2f", novoSalario));
        System.out.println("Aumento = "+String.format("%.2f", aumento));
        System.out.println("Porcentagem = "+porcentagem+"%");
        
        sc.close();
	}

}
